src/components/studies/app-design/LeadInvestigatorDropdown.tsx

```diff
@@ -107,43 +109,54 @@ const LeadInvestigatorDropdown: React.FunctionComponent<LeadInvestigatorDropdown
       setLeadInvestigatorOptions(leadInvestigatorArray)
     }
     getLeadResearchAccount()
-  }, [])
+  }, [token, orgMembership])
 
   return (
     <div>
       <Box marginLeft="8px">Lead Principle Investigator*</Box>
-      <div>
-        <button
-          className={clsx(
-            classes.selectPrincipleInvestigatorButton,
-            !isLeadInvestigatorDropdownOpen &&
-              classes.selectPrincipleInvestigatorButtonClosed,
-          )}
-          onClick={() => {
-            setIsLeadInvestigatorDropdownOpen(!isLeadInvestigatorDropdownOpen)
-          }}
+      <Select
+        labelId="lead-investigator-drop-down"
+        id="lead-investigator-drop-down"
+        value={currentInvestigatorSelected || 'Select pricinple investigator'}
+        onChange={e => {
+          onChange(e.target.value)
+        }}
+        className={classes.container}
+        disableUnderline
+        classes={{
+          selectMenu: classes.selectMenu,
+          root: classes.selectPrincipleInvestigatorButton,
+        }}
+        MenuProps={{
+          classes: { list: classes.listPadding, paper: classes.listBorder },
```

Comment:
I don't know if you can, but if it is possible to just have a style property here rather than class, since it is only a single prop (border, padding) it would be ideal. I